### PR TITLE
[BZ:999846] Add jbossweb and jbossxacml to -with-security bom

### PR DESCRIPTION
**Pull request by gaol**
Bugzilla reference: https://bugzilla.redhat.com/show_bug.cgi?id=999846

Both jbossweb and jbossxacml need to be added to -with-security bom, in case the customer's pom depends on each declared dependencies in the -with-security bom.

### CODE REVIEW

**Comment by rafabene** [comment]
@pedroigor Are you ok with this ?

File: jboss-javaee-6.0-with-security/pom.xml [hunk 1]
```diff
@@ -70,6 +70,11 @@
                 <version>${version.org.jboss.security.negotiation}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
```
Comment (rafabene) [inline jboss-javaee-6.0-with-security/pom.xml:73]: Shouldn't it be provided ?

**Comment by pedroigor** [comment]
It is fine.

**Comment by rafabene** [comment]
@gaol After discussed with @pedroigor I think it's better to mark jbossweb and jbossxacml as provided.

Thanks

**Comment by pedroigor** [comment]
All those dependencies are provided by EAP via modules. We don't need them inside deployments as they must always follow the versions from EAP.

**Comment by rafabene** [comment]
Replaced by #47

**Pull request closed without merging**